play/install

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+ACTIVATOR_VERSION="1.3.2-minimal"
+
+/var/lib/tsuru/base/install
+
+apt-get install software-properties-common -y
```

Comment:
You'll need to add `unzip` here. Testing this I found that it's not available by default.